frontend/js/services/steps.js

```diff
@@ -79,6 +106,36 @@ angular
         return steps.views[steps.current].template + cacheKiller;
     };
 
+    steps.changeTab = (to) => {
+        if (to === tabs.current) {
+            return;
+        }
+        if(tabs.isCollapsed){
+            let sidebar = document.getElementById('main-sidebar');
```

Comment:
document.getElementById sollte in Angular vermeiden werden. Stattdessen lieber mit ng-Style oder ng-class arbeiten